tensorbay/label/attributes.py

```diff
@@ -94,8 +94,8 @@ class Items(ReprMixin, EqMixin):
         items: The items inside array type attributes.
 
     Raises:
-        TypeError: When both enum and type_ are absent or
-            when type_ is array and items is absent.
+        TypeError: When both ``enum`` and ``type_`` are absent or
```

Comment:
is that possible to find all places needs to add inline code block, and change them in one commit